.github/workflows/publish-model-2-packages.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish org.lflang to GitHub Packages
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
```

Comment:
I think this means it will also be triggered by the nightly build. Was this the intent? I'm thinking we should probably exclude `nightly`.